src/core/lender/tranche/operator/base.sol

```diff
@@ -16,34 +16,28 @@
 pragma solidity >=0.4.24;
 
 import "ds-note/note.sol";
-import "ds-math/math.sol";
+import "tinlake-math/math.sol";
 
 contract TrancheLike {
     function supply(address usr, uint currencyAmount, uint tokenAmount) public;
-    function redeem(address usr, uint currencyAmount, uint tokenAmount ) public;
+    function redeem(address usr, uint currencyAmount, uint tokenAmount) public;
 }
 
 contract AssessorLike {
     function calcTokenPrice() public returns(uint);
 }
-
-// Tranche
-contract Operator is DSNote,DSMath {
+// Abstract Contract
+contract BaseOperator is DSNote, Math {
```

Comment:
This contract doesn't need math I think. I'd add it to AllowanceOperator

Reply:
the contract uses `rdiv` :)

Reply:
Where does it use it? The baseoperator doesn't, just the AllowanceOperator